include/dem/dem.h

```diff
@@ -43,7 +43,6 @@
 #include <iostream>
 #include <unordered_set>
 
-using namespace DEM;
```

Comment:
Seems like a bit of a random change? We don't really have this namespace enforced though...

Reply:
Nah it's ok, if it's not used it's not used even if it is orthogonal to that PR.